buildSrc/src/main/kotlin/Config.kt

```diff
@@ -1,33 +1,33 @@
 object Config {
     object Versions {
         object Kotlin {
-            const val kotlin = "1.3.72"
+            const val kotlin = "1.7.22"
         }
         object Plugin {
-            const val androidGradle = "3.6.1"
-            const val publish = "0.11.1"
+            const val androidGradle = "7.3.1"
+            const val publish = "0.22.0"
```

Comment:
I see you migrated the logic to maven-publish, this one can be removed from dependencies then

Reply:
It should still use gradle-maven-publish-plugin, with an updated config syntax.
The POM probably needs to be updated with some details.
https://vanniktech.github.io/gradle-maven-publish-plugin/other/